### DISCUSSION TITLE
Rollup `emitFile()` in sveltekit library mode to `dist/`

### DISCUSSION THREAD

**Discussion by benjaminpreiss** [sveltejs/kit#10999] [Q&A]
When using rollups `emitFile()` in sveltekit library mode, the emitted file is emitted into the `/.svelte-kit` folder. Does anyone know how I can emit the file in the `/dist` folder?

**Comment by teemingc** [comment]
You can change the config in svelte.config.js to output to a different directory name

https://kit.svelte.dev/docs/configuration#outdir

Or append your script with a move file command

```bash
svelte-package && mv .svelte-kit/file dist/
```

**Comment by benmccann** [comment]
Which adapter are you using? Each have a final output destination with most being `build`. The adapter should copy all necessary files from the intermediate `.svelte-kit` output destination to the final output destination.

**Reply by benjaminpreiss** [reply]
That is a good question - I think the default adapter / I havent specified an adapter...

But the files created with `emitFile` did not land in the `/dist` folder but only in the `/.svelte-kit` folder.

**Reply by benmccann** [reply]
There is no `/dist` folder by default. The default adapter is `adapter-auto`, which will attempt to detect which hosting platform you're deploying to and build the appropriate output. It's possible that a `/dist` folder is generated on one of those platforms, but I'd need to know which one. You won't have a `/dist` folder locally with `adapter-auto`
